src/main/java/bio/terra/common/Table.java

```diff
@@ -9,6 +9,7 @@
     UUID getId();
     String getName();
     List<Column> getColumns();
+    Long getRowCount();
```

Comment:
Is this the point where we need to split snapshot table class from regular table class?